src/Mod/Part/Gui/ViewProvider2DObject.h

```diff
@@ -67,14 +67,16 @@ class PartGuiExport ViewProvider2DObjectGrid : public ViewProvider2DObject
     App::PropertyEnumeration GridStyle;
     App::PropertyBool TightGrid;
     App::PropertyBool GridSnap;
-    App::PropertyBool GridAutoSize;
+    App::PropertyBool GridAuto; //Change size of grid based on view area.
+    App::PropertyBool GridAutoSize; //TODO: Find if this has any use.

```

Comment:
If you don't know yet if this property is useful then just omit it.

Reply:
This is what I have done, but thought it would be best to check if this is not something that should be removed. As name is very similar, I thought it may have been an attempt for grid auto resize which has  been abandonned.